rdr_service/dao/questionnaire_response_dao.py

```diff
@@ -931,6 +939,18 @@ def _update_participant_summary(
 
         dna_program_consent_update_code = config.getSettingJson(config.DNA_PROGRAM_CONSENT_UPDATE_CODE, None)
 
+        if 'verified' in remote_id_info.keys() and 'verified_on' in remote_id_info.keys():
+            if remote_id_info['verified'] == 1:
+                # is the participant origin always going to be the same? if not, where can I pull this from
+                participant_summary.remoteIdVerificationOrigin = participant_summary.participantOrigin
+                participant_summary.remoteIdVerificationStatus = RemoteIdVerificationStatus.TRUE
+                participant_summary.remoteIdVerifiedOn = remote_id_info['verified_on']
+        elif 'verified' in remote_id_info.keys():
```

Comment:
You don't need to use .keys() here, the conditions in this if elif block can be simplified: 
```suggestion
        if 'verified' in remote_id_info and 'verified_on' in remote_id_info:
            if remote_id_info['verified'] == 1:
                # is the participant origin always going to be the same? if not, where can I pull this from
                participant_summary.remoteIdVerificationOrigin = participant_summary.participantOrigin
                participant_summary.remoteIdVerificationStatus = RemoteIdVerificationStatus.TRUE
                participant_summary.remoteIdVerifiedOn = remote_id_info['verified_on']
        elif 'verified' in remote_id_info:
```